rn-dictionary/src/nouns.rs

```diff
@@ -1006,7 +1006,7 @@ pub const WORDS: [&'static str; 4096] = [
     "lacker",
     "stemson",
     "grampus",
-    "hasn't",
+    "coati",
```

Comment:
Is this just because of the tick mark?

Reply:
I just don't think `hasn't` is a noun.. seemed out of place